WordPress/src/main/AndroidManifest.xml

```diff
@@ -577,7 +577,7 @@
             android:theme="@style/WordPress.NoActionBar" />
 
         <activity
-            android:name=".ui.photopicker.mediapicker.MediaPickerActivity"
+            android:name="org.wordpress.android.ui.mediapicker.MediaPickerActivity"
```

Comment:
I would consider to remove the base package `org.wordpress.android` for consistency with the other entries. wdyt?

Reply:
sure, this was added automatically by AS